pkg/build/pipelines/git-checkout.yaml

```diff
@@ -1,148 +1,181 @@
 name: Check out sources from git
-
 needs:
   packages:
     - git
-
 inputs:
   repository:
     description: |
       The repository to check out sources from.
     required: true
-
   destination:
     description: |
       The path to check out the sources to.
     default: .
-
   depth:
     description: |
       The depth to use when cloning. Set to -1 to not specify depth when cloning.
     default: 1
-
   branch:
     description: |
       The branch to check out, otherwise HEAD is checked out.
-
   tag:
     description: |
       The tag to check out.
-
   expected-commit:
     description: |
       The expected commit hash
-
   recurse-submodules:
     description: |
       Indicates whether --recurse-submodules should be passed to git clone.
     default: false
-
+  cherrypicks:
```

Comment:
I don't have a hugely strong opinion here, but git always hyphenates "cherry-pick", so I would expect this to be hyphenated as well.